.changeset/silly-hotels-greet.md

```diff
@@ -0,0 +1,5 @@
+---
+"@logto/translate": minor
```

Comment:
can you mark `@logto/cli` as well? also we should point out why we don't use major for this breaking change

Reply:
Yeah, updated